client/lib/components/page_scaffold/page_header.dart

```diff
@@ -38,14 +42,21 @@ class PageHeader extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final titleWrapper = Padding(
-      padding: showBackButton
-          ? EdgeInsets.zero
-          : EdgeInsets.symmetric(horizontal: 8),
-      child: Text(
-        title,
-        style: TextStyle(color: Constants.accentNavyColor),
-      ),
-    );
+        padding: showBackButton
+            ? EdgeInsets.zero
+            : EdgeInsets.symmetric(horizontal: 8),
+        child: Column(
+          children: [
+            title != null
+                ? Text(
+                    title,
+                    style: TextStyle(color: Constants.accentNavyColor),
+                  )
+                : Container(
+                    child: headerWidget,
+                  ),
```

Comment:
Delete unnecessary container

```suggestion
                : headerWidget,
```